### PR TITLE
Fix #37155: Simplify quick add menu icon and circle positioning

### PR DESCRIPTION
**Pull request by wielinde**
https://community.openproject.org/wp/37155

### CODE REVIEW

File: frontend/src/app/modules/common/header/quick-add-menu.sass [hunk 1]
```diff
@@ -2,33 +2,20 @@
...
-      border-width: 2px
-      border-style: solid
-      border-color: var(--button--alt-highlight-background-color)
-      border-radius: 50%
-      padding: 0.25rem
-      width: 26px
-      height: 26px
-      content: ''
-      position: absolute
+  .op-quick-add-menu--button
```
Comment (b12f) [inline frontend/src/app/modules/common/header/quick-add-menu.sass]: this increases the specificity of the selector by one class, I'd really prefer if we stick to `&--button`. Is there a psecific reason for this change?

File: frontend/src/app/modules/common/header/quick-add-menu.sass [hunk 2]
```diff
@@ -2,33 +2,20 @@
...
+    > i::before
```
Comment (b12f) [inline frontend/src/app/modules/common/header/quick-add-menu.sass]: here you're also increasing specificity. If the styles were being overridden I'd like us to solve this at css causing the problem, not here.

**Comment by wielinde** [comment]
@b12f Thank you for the review. I just improved it a bit. Still I am not happy that I have to repeat the new icon class when the ancestor gets hovered.

**Comment by wielinde** [comment]
the failing feature spec has nothing to do with this PR.

File: frontend/src/app/modules/common/header/quick-add-menu.sass [hunk 3]
```diff
@@ -2,33 +2,19 @@
...
-      position: absolute
-
-    &:hover
-      > *
-        opacity: 0.9
-
-      &::after
-        opacity: 0.8
+  &:hover
+    .op-quick-add-menu--icon::before
```
Comment (b12f) [inline frontend/src/app/modules/common/header/quick-add-menu.sass]: You can shorten this with `&:hover #{&}--icon::before` as a selector, though I'm not sure if it's much more readable.

**Review by b12f — APPROVED** [review]
LGTM

File: frontend/src/app/modules/common/header/quick-add-menu.sass (hunk 3 above)
Comment (oliverguenther) [reply frontend/src/app/modules/common/header/quick-add-menu.sass]: You can also do

```
&:hover &
  &--icon:before
```

which matches a bit more the BEM syntax we usually use without interpolation

Comment (b12f) [reply frontend/src/app/modules/common/header/quick-add-menu.sass]: That's cool I hadn't thought of that yet :+1:

Comment (wielinde) [reply frontend/src/app/modules/common/header/quick-add-menu.sass]: Nice, I learned something today, I believe. Thank you both

**Pull request merged**